### DISCUSSION TITLE
How do i add a custom screen for fivem ?

### DISCUSSION THREAD

**Discussion by Diondro299d** [kibook/pmms#99] [Q&A]
Hey, I got this cinema MLO and would like to place a screen there, but I don't know how. 
![image_2024-08-09_123908384](https://github.com/user-attachments/assets/8f146379-9ae6-48e0-9f44-08ae7213d41e) so how do I go about getting started.

**Comment by SleepingFox25** [comment]
Hi, did you find a solution ? I'm trying to do the same thing with a default GTA V TV prop
